.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -67,6 +67,16 @@ body:
       description: This can be obtained with `reth db version`
     validations:
       required: false
+  - type: dropdown
+    id: node
+    attributes:
+      label: What type of node are you running?
+      options:
+        - Archive (default)
+        - Full via --full flag
+        - Pruned with custom reth.toml config
```

Comment:
should ask for what the full config, if custom pruned

Reply:
that can be added in additional context

Reply:
added